Separateinstall

astridx opened the pull request:
Pull Request for Issue #113.

### Summary of Changes
Insert the the method  public function runInstallAndTest into robofile.php

### Testing Instructions
Apply this changes and try the command 
tests/codeception/vendor/bin/robo run:install-and-test
Joomla will be installed and after that you can choose a test feature.


### Documentation Changes Required
The command 
tests/codeception/vendor/bin/robo run:install-and-test 
starts a new installation, disables Statistics, sets Error Reporting to Development and you can choose a test. I have updated Readme.md as well. Relevant after line 183.

astridx committed:
create a new method to separate install and after that start a feature

astridx committed:
add a methode in robofile.php for making it possible to run installation and after that choose a feature

astridx committed:
Duplicate code removed

yvesh commented on README.md:
````diff
@@ -180,7 +187,19 @@ If you want to see steps then you can use `--steps` option of codeception. Check
+
+**_Windows_**
+```cmd
+$ tests\codeception\vendor\bin\robo run:install-and-test
````
Hmm is the command not a bit long? :) How about we use run:test and you can add --install as parameter?

astridx replied:
I like it like it is. You do not need the whole command. If you type
`tests\codeception\vendor\bin\robo run:i`
the "installandtest" methode will be executed. 
But the long name explained exactly what the methode will do excetly.

yvesh commented on RoboFile.php:
```diff
@@ -389,27 +361,31 @@ public function runTests($opts = ['use-htaccess' => false, 'env' => 'desktop'])
-			->arg($this->testsPath . 'acceptance/frontend/')
-			->run()
-			->stopOnFail();
+                $this->loadFeatures($pathToTestFile,$suite);
```
Space after comma

astridx replied:
I have inserted a space :)
Next week I will reinstall my code styles ...

degobbis committed:
Composer update (#118)

Update selenium-server-standalone to version 3.0.1.2 for working on linux with chrome

astridx committed:
new branch

astridx committed:
Coding Standards

astridx committed:
Coding Standards2

astridx committed:
Coding Standards3

astridx commented:
I do not like this an more and there are also overtaken changes.
